assets/js/protocol/protocol-navigation.js

```diff
@@ -191,10 +231,10 @@ if (typeof window.Mzp === 'undefined') { // eslint-disable-line block-scoped-var
     Navigation.bindEvents = function() {
         _navItemsLists = document.querySelectorAll('.mzp-c-navigation-items');
         if (_navItemsLists.length > 0) {
-            var navButtons = document.querySelectorAll('.mzp-c-navigation-menu-button');
-            for (var i = 0; i < navButtons.length; i++) {
-                navButtons[i].addEventListener('click', Navigation.onClick, false);
-            }
+            _navMenuButtons = document.querySelectorAll('.mzp-c-navigation-menu-button');
+            _navMenuButtons.forEach(function (menuButton) {
```

Comment:
Issue (blocking) This throws an error in IE: `Object doesn't support property of method 'forEach'`, since in older browsers a NodeList is treated differently to a regular array. Sticking to a conventional `for` loop might be better here.

Reply:
Oh shoot, didn't realize that - thank you